dpuagent/ansible/dpuagent.py

```
@@ -0,0 +1,130 @@
+copy_arg.src = "%s/dpu-iptables" % file_root
+copy_arg.dest = "%s/dpu-iptables" % dpuagent_root
+copy_arg.args = "force=yes"
+copy(copy_arg, host_post_info)
+
+
+# name: install zstack-store
+if client == "false":
+    command = "bash %s %s %s" % (dest_pkg, fs_rootpath, max_capacity)
+else:
+    command = "bash " + dest_pkg
+run_remote_command(command, host_post_info)
```

Ruff (0.15.2)

[error] `run_remote_command` may be undefined, or defined from star imports

(F405)